Sources/Plasma/FeatureLib/pfPatcher/plManifests.cpp

```diff
@@ -54,22 +54,38 @@ Mead, WA   99021
 
 plFileName plManifest::ClientExecutable()
 {
+#ifdef HS_BUILD_FOR_MACOS
+    return MANIFEST("plClient.app", "UruExplorer.app");
+#else
     return MANIFEST("plClient.exe", "UruExplorer.exe");
+#endif
```

Comment:
Can we do a macro like `PL_EXECUTABLE_SUFFIX` and set it based on filesystem, and then do something like this? (Thinking about how this is going to need to eventually handle no extension for Linux executables)

```suggestion
    return MANIFEST("plClient" PL_EXECUTABLE_SUFFIX, "UruExplorer" PL_EXECUTABLE_SUFFIX);
```